portlets/src/main/frontend/package.json

```diff
@@ -51,6 +51,8 @@
     "vue-popperjs": "^2.2.0",
     "vue-table-component": "^1.9.2",
     "vue-tabs-with-active-line": "^1.1.4",
-    "vuejs-progress-bar": "^1.0.9"
+    "vuejs-progress-bar": "^1.0.9",
+    "echarts": "^4.2.1",
```

Comment:
[DEVSTD-29](https://docs.google.com/spreadsheets/d/1s-op62uot1dy4YwMA-JswxNg0DwS4sdmg1eAll0VHIU/edit#gid=0&range=A30) No third party library should be defined or imported using npm. 

In order to add this js file as third party library, you can ask an architect or TL and then add it in `gatein-resources.xml` as an AMD Module. ( cc @azayati FYI )